Resources/views/standard_layout.html.twig

```diff
@@ -205,47 +205,7 @@ file that was distributed with this source code.
-                                                    {% for admin in group.items if item_count == 0 %}
-                                                        {% if admin.hasroute('list') and admin.isGranted('LIST') %}
-                                                            {% set item_count = item_count+1 %}
-                                                        {% endif %}
-                                                    {% endfor %}
-                                                {% endif %}
-
-                                                {% if display and (item_count > 0) %}
-                                                    {% set active = false %}
-                                                    {% for admin in group.items %}
-                                                        {% if admin.hasroute('list') and admin.isGranted('LIST') and app.request.get('_sonata_admin') == admin.code %}
-                                                            {% set active = true %}
-                                                        {% endif %}
-                                                    {% endfor %}
-                                                    <li class="treeview{% if active %} active{% endif %}">
-                                                        <a href="#">
-                                                            {% if group.icon|default() %}{{ group.icon|raw }}{% endif %}
-                                                            <span>{{ group.label|trans({}, group.label_catalogue) }}</span>
-                                                            <i class="fa pull-right fa-angle-left"></i>
-                                                        </a>
-                                                        <ul class="treeview-menu{% if active %} active{% endif %}">
-                                                            {% for admin in group.items %}
-                                                                {% if admin.hasroute('list') and admin.isGranted('LIST') %}
-                                                                    <li{% if app.request.get('_sonata_admin') == admin.code %} class="active"{% endif %}><a href="{{ admin.generateUrl('list')}}"><i class="fa fa-angle-double-right"></i> {{ admin.label|trans({}, admin.translationdomain) }}</a></li>
-                                                                {% endif %}
-                                                            {% endfor %}
-                                                        </ul>
-                                                    </li>
-                                                {% endif %}
-                                            {% endfor %}
-                                        </ul>
+                                        {{ knp_menu_render(admin_pool.menu, {'template' : 'SonataAdminBundle:Menu:sonata_menu.html.twig'}) }}
```

Comment:
Cannot we make the SonataAdminBundle:Menu:sonata_menu.html.twig configurable like other template ?